Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -691,6 +695,116 @@ std::pair<float,float> HGCalDDDConstants::locateCellTrap(int lay, int irad,
   return std::make_pair(x,y);
 }
 
+bool HGCalDDDConstants::maskCell(const DetId& detId, int corners) const {
+  bool mask(false);
+  if (corners > 2 && corners < 6) {
+    if ((mode_ == HGCalGeometryMode::Hexagon8) ||
+	(mode_ == HGCalGeometryMode::Hexagon8Full)) {
+      int N(0), layer(0), waferU(0), waferV(0), u(0), v(0);
+      if (detId.det() == DetId::Forward) {
+	HFNoseDetId id(detId); 
+	N      = getUVMax(id.type());
+	layer  = id.layer();
+	waferU = id.waferU();
+	waferV = id.waferV();
+	u      = id.cellU();
+	v      = id.cellV();
+      } else {
+	HGCSiliconDetId id(detId);
+	N      = getUVMax(id.type());
+	layer  = id.layer();
+	waferU = id.waferU();
+	waferV = id.waferV();
+	u      = id.cellU();
+	v      = id.cellV();
```

Comment:
Indentation for lines 706-711 and 714-719 is missing.

Reply:
Please check again. The indentations are there

Reply:
The problem is the mixture of tabs and spaces. Many people have tabs set to 4 spaces rather than 8. It would be helpful if you could configure your editor not to use tabs as indentation.

Reply:
Indentation is handled now